src/python/build_env_setup.py

```diff
@@ -49,3 +49,5 @@
     if "_WIFI" in target_name:
         env.Replace(UPLOAD_PROTOCOL="custom")
         env.Replace(UPLOADCMD=upload_via_esp8266_backpack.on_upload)
+elif platform in ['espressif32'] and "_WIFI" in target_name:
+    env.Replace(UPLOADCMD=upload_via_esp8266_backpack.on_upload)
```

Comment:
Add add `env.Replace(UPLOAD_PROTOCOL="custom")` above this line, so the `build_flags.py` changes are not necessary.

Reply:
It's necessary, i have add env.Replace(UPLOAD_PROTOCOL="custom") above this line, it does not work, it still use esptool

Reply:
I have change this to section to
```
elif platform in ['espressif32'] and "_WIFI" in target_name:
    env.Replace(UPLOAD_PROTOCOL="custom")
    env.Replace(UPLOADCMD=upload_via_esp8266_backpack.on_upload)
```

and add
```
upload_port = 10.0.0.1
```
 to the end of the target definition in the diy_2400.ini file.i.e.
```
[env:DIY_2400_TX_ESP32_SX1280_E28_via_WIFI]
extends = env:DIY_2400_TX_ESP32_SX1280_E28_via_UART
upload_port = 10.0.0.1
```

and it all works.  I don't this build_flags is the right place to mucking about with upload stuff. That should be for building only.

Reply:
This does not work for esp backpack which using elrs_tx.local instead of 10.0.0.1

Reply:
For the DIY 2400 the backpack hardware is unused and you will be flashing the ESP32 directly over wifi.
You shouldn't have anything on the backpack ESP8266 on those systems yet as the only backpack code is teh one used to flash the STM32 based systems.

Reply:
You are right, have you made another pr with these code.

Reply:
If you update your code and push it will update the PR automatically.